README.md

```diff
@@ -29,11 +29,88 @@ composer require alleyinteractive/wp-rest-api-guard
 
 ## Usage
 
-Activate the plugin in WordPress and use it like so:
+The WordPress REST API is generally very public and can share a good deal of
+information with the internet anonymously. This plugin aims to make it easier to
+restrict access to the REST API for your WordPress site.
+
+### Settings Page
+
+The plugin can be configured via the Settings page (`Settings -> REST API
+Guard`) or via the relevant filter:
+
+<img width="1276" alt="Screen Shot 2022-10-06 at 4 24 01 PM" src="https://user-images.githubusercontent.com/346399/194411352-aa05e939-3fd1-4e37-a3d5-276c1c5c288f.png">
```

Comment:
Recommend putting in more descriptive alt text here. Could also be converted to Markdown image format.